config/initializers/10-patches.rb

```diff
@@ -93,9 +125,48 @@ def full_message(attribute, message)
                                :message   => message
                              })
     end
+
+    private
+
+    def error_symbols
+      @error_symbols ||= Hash.new
+    end
+
+    def writable_symbols_and_messages_for(attribute)
+      attribute = attribute.to_sym # avoid mixed symbol/string keys
+      error_symbols[attribute] = [] unless error_symbols[attribute]
+
+      error_symbols[attribute]
```

Comment:
I'm all for explicit, but this seems somewhat redundant to me. Isn't the whole method body identical to `error_symbols[attribute.to_sym] ||= []` ?